.github/CONTRIBUTING.md

```diff
@@ -72,6 +72,10 @@ $ gox "linux darwin" "386 amd64 arm" \
 # run other commands like tests
 ```
 
+### Updating the templates
+
+If you happen to update the provider templates (in `/templates`), you need to run `go generate` to update the `autogen` package.
```

Comment:
Another comment in the file mentions that the generation step is required for the web UI. If that's true, we should mention it there as well and possibly update the other comment.